### PR TITLE
test(logging): add `dateTime` action trigger

### PR DESCRIPTION
**Pull request by kdhttps**
ref https://github.com/GluuFederation/gluu-passport/issues/387

### CODE REVIEW

File: test/logging.spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const convertOneDigitToTwo = (number) => {
+  return ('0' + (number)).slice(-2)
+}
+
+describe('logging', () => {
+  it('should log currect datetime', async () => {
+    const passportLogFilePath = path.join(__dirname, '../server/utils/logs/passport.log')
+    const data = await fs.readFile(passportLogFilePath, 'binary')
+    const newDate = new Date()
+    // YYYY-MM-DD HH
```
Comment (christian-hawk) [inline test/logging.spec.js:16]: What is triggering the action? What is being logged?

**Review by christian-hawk — CHANGES_REQUESTED** [review]
It looks like you are repeating a commit from #365 that has already been merged. If not, can you please explain?

File: test/logging.spec.js (hunk 1 above)
Comment (kdhttps) [reply test/logging.spec.js:16]: I think you missed my comment https://github.com/GluuFederation/gluu-passport/issues/387#issuecomment-968694782

- [x] it should be an placed in an integration test, not an unit test
- [x] assert.include should be used, to ensure proper assertion messages from chai
- [ ] there should be an action trigger

The last one remains. could you please describe?

**Comment by christian-hawk** [comment]
@kdhttps could you please solve conflict?

File: test/logging.spec.js (hunk 1 above)
Comment (kdhttps) [reply test/logging.spec.js:17]: Done mate

**Comment by kdhttps** [comment]
> @kdhttps could you please solve conflict?

Done mate, please review!

**Pull request merged**